doc/content/the-things-stack/migrate-to-v3/migrate-using-console.md

```diff
@@ -5,7 +5,8 @@ weight: 100
 
 This section explains steps for migrating end devices from The Things Network V2 to The Things Stack Community Edition using <a href="https://www.thethingsindustries.com/docs/getting-started/console/" target="_blank">The Things Stack Community Edition Console</a>.
 
-{{< warning >}} Migrating active sessions using The Things Stack Community Edition Console is not available. {{</ warning >}}
+{{< note >}} Migrating active sessions using The Things Stack Community Edition Console is available since the latest update to V3.12. <a href="https://www.thethingsindustries.com/docs/getting-started/migrating/migrating-from-v2/" target="_blank">click here</a>.{{</ note >}}
```

Comment:
This refers to using **the Console** -> it is still not possible to migrate active session using the Console as far as I know

Reply:
It is, I tried it in the past.

Reply:
@neoaggelos can you ack?

Reply:
I once asked @neoaggelos this, and his answer was that you can indeed reuse some parameters (like session keys) when adding an end device using Console, but for it to be a **true** session migration, you would have to take many more parameters into consideration (frame counters, MAC state, Rx1 delay, ADR, ...). Having this many parameters to fill would clearly be too complicated, so this wasn't an option. 

Still, I'm really not sure and if that's really the case, we should update our documentation asap.

Reply:
Importing end devices via the Console is absolutely possible, using the `Import End Devices` form. It uses the same JSON file that can be imported with the CLI.